pageScraper.js

```diff
@@ -0,0 +1,59 @@
+const scraperObject = {
+  url: "https://www.reserve.unilodge.com.au/categoryInfo.html?category=273",
+  async scraper(browser) {
+    let page = await browser.newPage();
+    console.log(`Navigating to ${this.url}...`);
+    // Navigate to the selected page
+    await page.goto(this.url);
+    const data = [];
+    const rate = [];
+    // Wait for the required DOM to be rendered
+    await page.waitForSelector(".ul_rooms");
+    // Get the link to all the required books
+    let urls = await page.$$(".ul_room");
+    let Obj = await page.$$(".ul_stayPrice");
```

Comment:
why this is here?
It should be inside the loop. we don't need to select all the stay prices at once.
it should be at the room level

Reply:
Done